lib/ansiblelint/rules/SudoRule.py

```diff
@@ -15,10 +15,10 @@ def _check_value(self, play_frag):
         if isinstance(play_frag, dict):
             if 'sudo' in play_frag:
                 results.append(({'sudo': play_frag['sudo']},
-                                'Deprecated sudo feature'))
+                                'Deprecated sudo feature', play_frag['__line__']))
```

Comment:
This doesn't seem related to this PR.

Reply:
Oh, wait. I see. You modified it for testing. But does the test for this exist? I guess you should unremove the test and use this rule there.

Reply:
hmm, yes I think you're right.

Reply:
I think I just misunderstood what you meant, fixing